corehq/apps/repeaters/models.py

```diff
@@ -63,9 +63,9 @@ def simple_post_with_cached_timeout(data, url, expiry=60 * 60, force_send=False,
         raise RequestConnectionError(e.message)
 
     if not 200 <= resp.status_code < 300:
-        message = u'Status Code {}: {}'.format(resp.status_code, resp.reason)
+        message = u'Status Code {}: {}. {}'.format(resp.status_code, resp.reason, getattr(resp, 'content', None))
         cache.set(key, message, expiry)
-        raise RequestConnectionError(message)
```

Comment:
@benrudolph I think you added this... I removed it because a `404` isn't actually an `Exception`